swupd/format_switch_test.go

```diff
@@ -85,14 +69,79 @@ func TestManifestFormats25to26(t *testing.T) {
 	}
 	checkManifestContains(t, ts.Dir, "30", "test-bundle", expSubs...)
 	checkManifestContains(t, ts.Dir, "30", "MoM", "minversion:\t20")
+}
 
-	ts.addFile(40, "test-bundle", "/foo", contents+"D")
+// Iterative manifest support added in format 26
+func TestFormats25to26IterativeManifest(t *testing.T) {
+	ts := newTestSwupd(t, "format25to26iterativeManifest")
+	defer ts.cleanup()
+
+	ts.Bundles = []string{"test-bundle"}
+
+	// Format 25 should not have iterative manifest support
+	ts.Format = 25
+	ts.addFile(10, "test-bundle", "/foo", "content")
+	ts.createManifests(10)
+
+	ts.addFile(20, "test-bundle", "/foo", "new content")
+	ts.createManifests(20)
+	checkManifestContains(t, ts.Dir, "20", "MoM", "MANIFEST\t25")
+
+	// Iterative manifests should not have entries in the MoM or be generated
+	checkManifestNotContains(t, ts.Dir, "20", "MoM", "I...\t")
+	ts.checkNotExists("www/20/Manifest.test-bundle.I.10")
+	ts.checkNotExists("www/20/os-core.I.10")
```

Comment:
Needs to be `"www/20/Manifest.os-core.I.10"`, you are missing the "Manifest." part of the name.

Reply:
Thanks, I pushed the fix.